curvefs/proto/mds.proto

```diff
@@ -181,6 +214,9 @@ service MdsService {
     rpc AllocateS3Chunk(AllocateS3ChunkRequest) returns (AllocateS3ChunkResponse);
     rpc ListClusterFsInfo (ListClusterFsInfoRequest) returns (ListClusterFsInfoResponse);
 
+    rpc GetLatestTxId(GetLatestTxIdRequest) returns (GetLatestTxIdResponse);
+    rpc CommitTx(CommitTxRequest) returns (CommitTxResponse);
```

Comment:
if you move a rpc service, there will be a compatible issue.

Reply:
> if you move a rpc service, there will be a compatible issue.

I will keep both of these 2 methods for compatibility.